mp/src/game/client/momentum/ui/leaderboards/LeaderboardsTimes.h

```diff
@@ -75,13 +79,15 @@ class CLeaderboardsTimes : public vgui::EditablePanel
     // methods
     void FillLeaderboards(bool bFullUpdate);
     void SetPlaceColors(vgui::SectionedListPanel* panel, TimeType_t type) const;
-    void LoadLocalTimes(KeyValues *kv);
+    void LoadLocalTimes();
+    void LocalTimesCallback();
     void LoadOnlineTimes(TimeType_t type);
     void ConvertLocalTimes(KeyValues *pKv);
+    void LocalTimesVectorToLeaderboards();
     // Place vector times into leaderboards panel (sectionlist)
     void OnlineTimesVectorToLeaderboards(TimeType_t type);
 
-    bool GetPlayerTimes(KeyValues *outPlayerInfo, bool fullUpdate);
+    bool FillPlayerTimes(bool fullUpdate);
```

Comment:
Since there is now 4 local times related methods, try seeing if you can move some to be private. `ConvertLocalTimes` followed by a public `LocalTimesVectorToLeaderboards` is a little weird. Can some be combined, especially if it's only used by one function?